build_tooling/parallel_test.sh

```diff
@@ -28,7 +18,7 @@ catch=`{ which catchsegv 2>/dev/null || echo ; } | tail -n 1`
 
     export ARCTICDB_RAND_SEED=$RANDOM
 
-    $catch python -m pytest -n $splits $PYTEST_XDIST_MODE -v --log-file="$TEST_OUTPUT_DIR/pytest-logger.$group.log" \
+    $catch python -m pytest $PYTEST_XDIST_MODE -v --log-file="$TEST_OUTPUT_DIR/pytest-logger.$group.log" \
```

Comment:
Why do we still have XDIST_MODE, or the env var `PARALLEL_TEST_ROOT` ? In fact shouldn't we rename this file too?

Reply:
We might want to reinstate some parallelism later, but if not then yes, probably all of those things